src/containers/Main.js

```diff
@@ -44,8 +61,31 @@ class Main extends Component {
             );
           });
     }
+    handleSearchAPI(){
+        let textInput = document.getElementById('search');
+        let timeout = null;
+        textInput.onkeyup = (e) => {
+        clearTimeout(timeout);
+        timeout = setTimeout(() =>  {
+            axios.get(
+                "https://empresas.ioasys.com.br/api/v1/enterprises?&name=" + textInput.value,
+                { headers: { "Content-Type": "application/json",
+                "access-token": localStorage.getItem("userToken"),
+                "client": localStorage.getItem("userClient"),
+                'uid': localStorage.getItem("userID") } }
+              )
+              .then(response => {
+                this.setState({enterpriseList: response.data.enterprises})
+              })
+              .catch(function(error) {
+                console.error(
```

Comment:
Seria melhor você tratar essa mensagem de erro não? Não acho muito válido usar console em aplicações frontend. Seria bom você criar talvez um toast e mandar uma mensagem de erro nele, quando for mensagem de erro da API você exibe a mensagem e quando for mensagem de erro técnico você manda uma mensagem genérica para o usuário, geralmente as APIS tem padrões de corpo da mensagem para você poder diferenciar.